webapp-django/crashstats/signature/templates/signature/signature_report.html

```diff
@@ -145,8 +184,12 @@
 <script>
 var BASE_URL = location.protocol + '//' + location.host;
 var FIELDS = {{ fields | json_dumps }};
+var DEFAULT_CHANNEL = {{ channel | json_dumps }};
```

Comment:
Not a big deal but there's no need to using ``json_dumps`` here, since it's a simple string. You can just use ``... = '{{ channel }}';``.